### DISCUSSION TITLE
Remove AppImage and distribute with .deb and/or .rpm instead?

### DISCUSSION THREAD

**Discussion by anatawa12** [vrc-get/vrc-get#1533] [Ideas]
There are many issues related to AppImage like #1532, #934, #1346 and others so I'm considering to remove AppImage before v1 release now.

I usually don't use Linux so I want feedback from community before making decision.

If you're using linux, please let us know some your thoughts.

**Comment by Spokeek** [comment]
I don't use Unity on Linux myself and so don't use the Linux version of ALCOM, so i'm unsure if i would be a good reference for that.
But i like the idea of packaging the application for Linux, it is the good approach.

I heard that flatpack was good to package application cross distribution and to sandbox it to avoid creating specific versions for distributions.
On the personal side i only started recently to work on doing RPM packaging at work, so i can help a bit but not provide expertise.

**Reply by anatawa12** [reply]
Umm... I feel sandboxing would cause similar problem with appimage and unity so I don't like to use package managers with sandbox.

**Comment by KisaragiEffective** [comment]
それぞれ以下のような欠点があると認識しています。個人的にはAppImageの配布を継続してほしいですが、不具合が多いならprebuilt AppImageの配布をやめてもいいかもしれません。
- deb/rpm: 一般にはシステム領域への変更を伴う
- Flatpak: 半端な隔離のせいでトラブルシューティングが難しくなる

個人的にはflatpakやsnapにはあまりいい思い出がないので採用する価値を感じません。

**Comment by anatawa12** [comment]
I didn't decide if we'll remove AppImage in the future yet, but I'm going to add deb / rpm release since the next version.

**Reply by Spokeek** [reply]
Sounds good
